content/grants/dusko-valve.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Dusko Valve"
+date = "2022-04-15"
+
+[taxonomies]
+grant_type = [ "Proposal" ]
+grant_category = [ "App Dev", "Other" ]
+
+[extra]
+image = ""
+description = "a tool which allows NFT artists to offer planets for redemption to their patrons alongside their NFTs"
+reward = "$15000 in WSTR"
+assignee = "~tasset-norlux"
+champion = "~tacryt-socryp"
+grant_id = "P0143"
+completed = false
+canceled = false
+link = ""
++++
+
+## Purpose
+
+The building of Valve, a tool which allows NFT artists to offer planets for redemption to their patrons alongside their NFTs. I.e. I buy an NFT, I also get a planet. Initially, this would take the form of a contract which queries an NFT contract and whitelists planets on a corresponding webpage for those who purchased an NFT from that contract. It may evolve into a full fledged, no-code marketplace that provides NFT artists the ability to offer planets alongside their NFTs.
+
+## Background
+
+While NFTs can be many things, they function primarily as a membership pass, an access card to a community. Having a tool which allows artists to deploy ships alongside their NFTs would draw NFT projects to Urbit as the platform for interacting with their community. It would also onboard more users to Urbit in the process. Further, many smaller NFT projects typically have higher quality communities and art which would draw artists rather than collectors to the Urbit ecosystem.
+
+Artists such as Haleek Maul have created custom collections of Urbit sigils which auctioned off both the artwork and corresponding planet or star together. This was a great example of artists selling artwork that immediately grants access to the Urbit community. Making a tool to do this at scale would further Urbit’s goal of becoming the primary social space for web3, as well as promote Urbit as an attractive, alternative platform to smaller NFT artists who don’t want to operate and tend to Discords, but are still interested and excited by the notion of communicating more directly with their supporters.
+
+## Implementation
+
+Money from this grant would go to creating a tool for NFT artists selling artwork to provide their community immediate access to Urbit by redeeming a planet.
+This would initially be an integration with Lit Protocol which queries an artist’s NFT contract to provide access to a modified planettine-based react app for distributing planets.
+
+## Stages of Valve
+
+1. The Valve MVP will be a redeployment of Tlon’s planettine distribution tool behind a Lit Protocol authentication layer. Planet slugs will be maintained in an airtable to be available for redemption through planettine’s react app. Tlon will work with me to polish onboarding copy for the frontend which will clearly explain how an NFT holder redeems, then boots up, their planet. Tlon will also host all the planets of the Dusko community, starting with the core team.
```

Comment:
*planentine. Also, given you're no longer using Lit protocol, please specify that you'll be using a lighter metamask-based auth.